src/ResourceManagement/NotificationHubs/NotificationHubs.Tests/project.json

```diff
@@ -7,7 +7,7 @@
     "summary": "NotificationHubs.Tests Tests.",
     "tags": [ "" ],
     "projectUrl": "https://github.com/Azure/azure-sdk-for-net",
-    "licenseUrl": "https://raw.githubusercontent.com/Microsoft/dotnet/master/LICENSE",
+    "licenseUrl": "https://raw.githubusercontent.com/Microsoft/dotnet/master/LICENSE"
   },
   "buildOptions": {
```

Comment:
Note that this contains breaking changes, you will need to update your version accordingly (if the library is not -preview or -prerelease)